src/test/java/com/microsoft/sqlserver/jdbc/tvp/TVPTypesTest.java

```diff
@@ -1,589 +1,589 @@
-
-        pstmt.execute();
-
-        rs = conn.createStatement().executeQuery("select * from " + table);
-        while (rs.next()) {
-            assertEquals(((SQLServerResultSet) rs).getSmallDateTime(1), returnValue);
-        }
-        if (null != pstmt) {
-            pstmt.close();
-        }
-    }
-    
-    @BeforeEach
-    private void testSetup() throws SQLException {
-        conn = DriverManager.getConnection(connectionString);
-        stmt = conn.createStatement();
-
-        dropProcedure();
-        dropTables();
-        dropTVPS();
-    }
-
-    @AfterAll
-    public static void terminate() throws SQLException {
-        conn = DriverManager.getConnection(connectionString);
-        stmt = conn.createStatement();
-        dropProcedure();
-        dropTables();
-        dropTVPS();
-    }
-
-    private static void dropProcedure() throws SQLException {
-        String sql = " IF EXISTS (select * from sysobjects where id = object_id(N'" + procedureName + "') and OBJECTPROPERTY(id, N'IsProcedure') = 1)"
-                + " DROP PROCEDURE " + procedureName;
-        stmt.execute(sql);
-    }
-
-    private static void dropTables() throws SQLException {
-        stmt.executeUpdate("if object_id('" + table + "','U') is not null" + " drop table " + table);
-    }
-
-    private static void dropTVPS() throws SQLException {
-        stmt.executeUpdate("IF EXISTS (SELECT * FROM sys.types WHERE is_table_type = 1 AND name = '" + tvpName + "') " + " drop type " + tvpName);
-    }
-
-    private static void createPreocedure() throws SQLException {
-        String sql = "CREATE PROCEDURE " + procedureName + " @InputData " + tvpName + " READONLY " + " AS " + " BEGIN " + " INSERT INTO " + table
-                + " SELECT * FROM @InputData" + " END";
-
-        stmt.execute(sql);
-    }
-
-    private void createTables(String colType) throws SQLException {
-        String sql = "create table " + table + " (c1 " + colType + " null);";
-        stmt.execute(sql);
-    }
-
-    private void createTVPS(String colType) throws SQLException {
-        String TVPCreateCmd = "CREATE TYPE " + tvpName + " as table (c1 " + colType + " null)";
-        stmt.executeUpdate(TVPCreateCmd);
-    }
-
-    private boolean parseByte(byte[] expectedData,
-            byte[] retrieved) {
-        assertTrue(Arrays.equals(expectedData, Arrays.copyOf(retrieved, expectedData.length)), " unexpected BINARY value, expected");
-        for (int i = expectedData.length; i < retrieved.length; i++) {
-            assertTrue(0 == retrieved[i], "unexpected data BINARY");
-        }
-        return true;
-    }
-
-    @AfterEach
-    private void terminateVariation() throws SQLException {
-        if (null != conn) {
-            conn.close();
-        }
-        if (null != stmt) {
-            stmt.close();
-        }
-        if (null != rs) {
-            rs.close();
-        }
-        if (null != tvp) {
-            tvp.clear();
-        }
-    }
-
+/*

```

Comment:
why did this whole file change? is it formatting only?

Reply:
I'll revert the formatting - unnecessary here

Reply:
Somehow GitHub is doing unnecessary comparisons, I compared the file to dev branch's and there's no formatting change actually. The only change is in changing private Junit test functions to public.